components/blitz/src/ome/services/blitz/repo/CheckedPath.java

```diff
@@ -454,4 +454,24 @@ public String toString() {
         return ofile;
     }
 
+    /**
+     * {@inheritDoc}
+     * Instances are equal if their string representations match.
+     */
+    @Override
+    public boolean equals(Object object) {
+        if (this == object)
+            return true;
+        if (!(object instanceof CheckedPath))
+            return false;
+        return this.fsFile.equals(((CheckedPath) object).fsFile);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public int hashCode() {
+        return this.fsFile.hashCode();
```

Comment:
might want to fiddle with the resulting number a bit so that it doesn't hash to the very same ?  (admittedly unlikely that both would be in the same collection...)